engineering/ember.md

```diff
@@ -41,8 +41,4 @@ refactoring. Components are also easier to test.
 
 ### [ember-test-selectors](https://github.com/simplabs/ember-test-selectors)
```

Comment:
Should we also remove this too?

Reply:
We do still use this, just not the test selector helper thing for tests. We still use it so that it strips out the test selectors for prod